gatsby-node.esm.js

```diff
@@ -215,6 +215,20 @@ async function sourcePrimerRailsData({actions, createNodeId, createContentDigest
   }
 }
 
+const reactIdMap = {
```

Comment:
So do we have to manually handle the data differences between the draft component and the "old" component?

Reply:
Sort of. This map just helps unify all the components under one name so we can render a status dropdown.